apps/sms/views/conversation/js/compose.js

```diff
@@ -733,7 +733,14 @@ var Compose = (function() {
           dom.message.removeChild(currentAttachmentDOM);
           disposeAttachmentNode(currentAttachmentDOM);
           onContentChanged(newAttachment);
-        }, this._onAttachmentRequestError);
+        }, (err) => {
+          var errId = err instanceof DOMError ? err.name : err.message;
```

Comment:
I changed the implementation to set the hideOnAction per item, that menu will not hide after method triggered when hideOnAction is true.